src/ducks/balance/helpers.js

```diff
@@ -0,0 +1,59 @@
+import groupBy from 'lodash/groupBy'
```

Comment:
It would import the entire lodash and the entire date-fns, no ? We use the [babel-plugin-lodash](https://github.com/lodash/babel-plugin-lodash) to prevent that for lodash, but we don't use [babel-plugin-date-fns](https://github.com/date-fns/babel-plugin-date-fns). I think we should keep the imports ungrouped or group them and use the plugin. What do you prefer ?

Reply:
I prefer use `babel-plugin-date-fns` ;)